src/reuse/helper/errorHandler.ts

```diff
@@ -21,19 +34,19 @@ export default class ErrorHandler implements IErrorHandler {
     this.logStackTrace = logStackTrace;
   }
 
-  public logException(errorObject: Error): Promise<never> {
+  public logException(errorObject: Error): never {
     if (errorObject) {
       let functionName = this._retrieveFunctionNameFromStack(errorObject);
 
-      const stackTrace = this.logStackTrace === true ? this._getFormattedStackTrace(errorObject) : "";
+      const displayStack = this.logStackTrace === true ? true : false;
```

Comment:
since logStackTrace is of type boolean as well, you can directly assign it 
```suggestion
      const displayStack = this.logStackTrace;
```
or even use the this.logStackTrace property directly in the function calls below

Reply:
added this.logStackTrace property to function call itself.